app/src/main/java/org/fossasia/openevent/general/auth/SignUpFragment.kt

```diff
@@ -0,0 +1,76 @@
+package org.fossasia.openevent.general.auth
+
+import android.support.v4.app.Fragment
+import android.arch.lifecycle.Observer
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import kotlinx.android.synthetic.main.fragment_signup.*
+import kotlinx.android.synthetic.main.fragment_signup.view.*
+import org.fossasia.openevent.general.MainActivity
+import org.fossasia.openevent.general.R
+import org.koin.android.architecture.ext.viewModel
+
+class SignUpFragment : Fragment() {
+
+    private val signUpActivityViewModel by viewModel<SignUpFragmentViewModel>()
+    private lateinit var rootView: View
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
+                              savedInstanceState: Bundle?): View? {
+        rootView = inflater.inflate(R.layout.fragment_signup, container, false)
+        lateinit var emailLogin: String
+        lateinit var passwordLogin: String
+        lateinit var confirmPassword: String
```

Comment:
Why?

Reply:
Actually I require emailLogin and passwordLogin for the login function in viewmodel

Reply:
Why can't it be stored locally?